.travis.yml

```diff
@@ -22,7 +22,7 @@ addons:
       - percona-toolkit
 
 install:
-  - go get -u github.com/Masterminds/glide
+  - curl https://glide.sh/get | sh
```

Comment:
`The command "curl https://glide.sh/get | sh" failed and exited with 1 during .` from travis